src/Data/List/Membership/Propositional/Properties.agda

```diff
@@ -179,6 +179,44 @@ module _ {p} {P : A → Set p} (P? : Decidable P) where
   ∈-filter⁻ : ∀ {v xs} → v ∈ filter P? xs → v ∈ xs × P v
   ∈-filter⁻ = Membershipₛ.∈-filter⁻ (P.setoid A) P? (P.subst P)
 
+------------------------------------------------------------------------
+-- nub and dedup
+
+module _ {ℓ} {R : Rel A ℓ} (R? : B.Decidable R) where
```

Comment:
Any chance you could generalise these results to setoid membership and then simply re-export them here, as is done throughout the rest of the file?